src/Service/DataFakerService.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/**
+ * @file
+ * Contains ElasticsearchFaker that creates test data in Elasticsearch.
+ */
+
+namespace App\Service;
+
+use Doctrine\ODM\MongoDB\DocumentManager;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * Class ElasticsearchFaker.
+ */
+class DataFakerService
```

Comment:
Should this service have been in a test namespace.

Reply:
Fixed